internal/backends/nodejs/nodejs.go

```diff
@@ -372,6 +372,10 @@ var nodejsGuessRegexps = util.Regexps([]string{
 	`(?m)(?:require|import)\s*\(\s*['"]([^'"{}]+)['"]\s*\)`,
 })
 
+var nodeIgnorePathSegments = map[string]bool{
+	"node_modules": true,
+}
+
```

Comment:
Nice, I wonder if should load `.gitignore` and/or hardcode some of these folders https://github.com/github/gitignore/blob/main/Node.gitignore, and maybe ignore all directories that start with `.`?

Reply:
Yeah, I'd like to do that in a separate PR if you don't mind